application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -109,9 +110,11 @@ public void onSlashCommand(@NotNull final SlashCommandEvent event) {
      */
     @NotNull
     private String convertInlineLatexToFull(@NotNull String latex) {
+        if (isInvalidInlineFormat(latex)) {
+            throw new ParseException("One dollar sign is invalid, wrap whole expression with it. ");
```

Comment:
The error message is not accurate. For example, if you give 3 dollars, it also triggers. But the message only says "one dollar".

> The amount of $-symbols must be divisible by two. Did you forget to close an expression?